.circleci/packaging/python/LICENSE

```diff
@@ -0,0 +1,201 @@
+                            Apache License
```

Comment:
We never put the whole Apache license in sources, just the very last block:

```
   Copyright [yyyy] [name of copyright owner]

   Licensed under the Apache License, Version 2.0 (the "License");
   you may not use this file except in compliance with the License.
   You may obtain a copy of the License at

       http://www.apache.org/licenses/LICENSE-2.0

   Unless required by applicable law or agreed to in writing, software
   distributed under the License is distributed on an "AS IS" BASIS,
   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   See the License for the specific language governing permissions and
   limitations under the License.
```

Reply:
Done.

Reply:
I just copied the LICENSE file from the current latest tarball on PyPi. Adding the current year and Google LLC.